gradio/routes.py

```diff
@@ -1184,6 +1184,10 @@ def robots_txt():
 
         @app.get("/monitoring", dependencies=[Depends(login_check)])
         async def analytics_login(request: fastapi.Request):
+            if not blocks.enable_monitoring:
```

Comment:
can we add a unit test for this?

Reply:
to test what? that the route is blocked?

Reply:
Yea